docs/src/css/custom.css

```diff
@@ -34,6 +34,10 @@ footer {
   --ifm-footer-background-color: #35364f !important;
   --ifm-code-background: var(--neutral-light-9);
   --docsearch-searchbox-background: var(--white);
+  --ifm-color-danger-contrast-background: #d0021b;
```

Comment:
should we have var colors for these?

Reply:
Yes! We did have the one for red and I updated the one for green. Good call